packages/kotti-single-select/src/SingleSelect.vue

```diff
@@ -68,8 +85,24 @@ export default {
 			queryString: '',
 			selectedLabel: '',
 			visible: false,
+			formOptions: null,
+			formOptionsContainer: null,
+			input: null,
+			formOptionStyle: '',
 		}
 	},
+	mounted() {
```

Comment:
This doesn’t handle scrolling, adding elements that move the component, etc..

Reply:
For example, you can verify this by just deleting something in your debugger:
<img width="927" alt="image" src="https://user-images.githubusercontent.com/1133858/58432360-35673100-80b2-11e9-8aad-3b878b122653.png">

Please use a library for this, as correctly implementing this behavior is insanely complicated.

Reply:
It is actually not, and I fixed it.

Reply:
@FlorianWendelborn has a point, there will definitely be some weird IE edge cases, this is why I was under the impression that popper handles this, even if it doesn't render in the body you can make it go into the body

Reply:
Please give me real cases because I cannot fix it otherwise. I will improve the popover and use it in the select in the future, but that is an improvement and I cannot do it right now.